<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>brush_dock</class>
 <widget class="QDockWidget" name="brush_dock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>94</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Brush Selection</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="spray_button">
      <property name="text">
       <string>Brush Size</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="brush_size">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>40</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="paint_button">
      <property name="text">
       <string>Paint</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">paint_button_group</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="erase_button">
      <property name="toolTip">
       <string>erase</string>
      </property>
      <property name="text">
       <string>Erase</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">paint_button_group</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_3">
      <property name="text">
       <string>Spray</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">paint_button_group</string>
      </attribute>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="paint_button_group"/>
 </buttongroups>
</ui>
